index.html

```diff
@@ -46,7 +46,7 @@
 					<div class="social-media module">
 						<h1>Follow us</h1>
 						<div class="follow-nature">
-							<ul>
+							<ul class="cleared">
```

Comment:
the css stylesheet that the code test came with had a class called 'cleared' which is being used as a "clearfix" (do some tutorials on clearfix if you dont know what it is). i needed to use a clearfix on this `<ul>` because inside of it you have floated elements and the parent element (the `<ul>`) will not surround the floated elements correctly if i set `overflow: visible` on the `<ul>` which is something i needed to do to complete this (as you'll see later in comments)